packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -2231,6 +2312,16 @@ class RoundSliderThumbShape extends SliderComponentShape {
   final double disabledThumbRadius;
   double get _disabledThumbRadius =>  disabledThumbRadius ?? enabledThumbRadius;
 
+  /// The resting elevation adds shadow to the unpressed thumb.
+  ///
+  /// The default is 1.
```

Comment:
Maybe the value should be asserted to be >= 0?

You could add something about using 0 for no shadow, 16 (or whatever) for an unusually big shadow. To give developers an idea about what the value means.

Here and elsewhere.